include/ck/tensor_operation/gpu/device/device_avgpool_bwd.hpp

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <vector>
+
+#include "ck/tensor_operation/gpu/device/device_base.hpp"
+#include "ck/utility/reduction_enums.hpp"
```

Comment:
This including is not needed !

Reply:
f72ad5e9542e22f7e9e31f4bb7c5796e4360c889